app/web/features/communities/events/constants.ts

```diff
@@ -1 +0,0 @@
-export const getExtraAvatarCountText = (count: number) => `+${count}`;
```

Comment:
This is technically unrelated, but it's not being used anymore since Ricardo's PR it seems